src/main/java/com/blackducksoftware/integration/jira/config/JiraConfigDeserializer.java

```diff
@@ -44,14 +47,23 @@ public BlackDuckJiraConfigSerializable deserializeConfig(final PluginIssueCreati
         final BlackDuckJiraConfigSerializable config = new BlackDuckJiraConfigSerializable();
         config.setHubProjectMappingsJson(projectMapping.getMappingsJson());
         config.setPolicyRulesJson(ticketCriteria.getPolicyRulesJson());
+
         logger.debug("Mappings:");
-        for (final BlackDuckProjectMapping mapping : config.getHubProjectMappings()) {
-            logger.debug(mapping.toString());
+        final Set<BlackDuckProjectMapping> projectMappings = config.getHubProjectMappings();
+        if (null != projectMappings) {
+            for (final BlackDuckProjectMapping mapping : projectMappings) {
+                logger.debug(mapping.toString());
+            }
```

Comment:
Is BlackDuckProjectMapping not stringable? Do we specifically want to print separate debugs messages for each of these?